lms/djangoapps/instructor_task/tests/test_tasks_helper.py

```diff
@@ -695,7 +696,87 @@ def test_cohort_content(self):
             self.assertDictContainsSubset(
                 {'action_name': 'graded', 'attempted': 4, 'succeeded': 4, 'failed': 0}, result
             )
+        problem_names = [u'Homework 1: Problem - Pröblem0', u'Homework 1: Problem - Pröblem1']
+        header_row = [u'Student ID', u'Email', u'Username', u'Final Grade']
+        for problem in problem_names:
+            header_row += [problem + ' (Earned)', problem + ' (Possible)']
 
+        self.verify_rows_in_csv([
```

Comment:
This isn't quite what I was thinking. Rather writing the same assertion as is on 785, writing a helper method is the right approach, IMO. I just disagreed with the helper method statically calling one assertion. It could have been further parameterized by accepting grade data and user info in order to be more re-usable.